src/helpers.rs

```diff
@@ -1,6 +1,3 @@
-#![allow(dead_code)]
```

Comment:
Just leave the allow statement there. The helpers are often unused in some feature permutation. I think we need to re-do the features at some point...